index.html

```diff
@@ -4,7 +4,71 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="style.css" rel="stylesheet" />
     <title>Personal Website</title>
   </head>
-  <body></body>
+
+  <body>
+    <header>
+      <h1>Adriana Morales Tremolada</h1>
+      <h2>Human</h2>
+      <nav>
+        <ul>
+          <li>About me</li>
+          <li>Interests</li>
+          <li>What the future holds</li>
+          <li>Contact</li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section>
+        <h2>About me</h2>
+      </section>
+      <section>
+        <h2>Interests</h2>
+      </section>
+      <section>
+        <h2>What the future holds</h2>
+      </section>
+      <section>
+        <h2>Contact</h2>
+      </section>
+    </main>
+
+    <footer>
+      <div>
+        <ul>
+          <li>About me</li>
+          <li>Interests</li>
+          <li>What the future holds</li>
+          <li>Contact</li>
+        </ul>
+      </div>
+      <div>
```

Comment:
A`section` would be better as the `div`. `div` is semantically meaningless for a screen reader. It should only be here for styling purpose.

Reply:
I used it for styling purposes afterwards, but you are right, the section is also a container. I will change it, tks!